src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -57,9 +59,26 @@ const PopupDrawingToolbar = ({
     reference,
 }: Props): JSX.Element => {
     const intl = useIntl();
+    const popupRef = React.useRef<PopupBase>(null);
+    const refRect = reference.getBoundingClientRect();
```

Comment:
Since this is a drawing-specific component, could we re-render based on when `drawnPathGroups` in the drawing reducer changes? That would allow us to avoid hitting the DOM every time this component renders.

Reply:
Good call, I'll use that instead.